picard/ui/mainwindow.py

```diff
@@ -104,13 +105,15 @@ class MainWindow(QtWidgets.QMainWindow, PreserveGeometry):
         config.BoolOption("persist", "view_toolbar", True),
         config.BoolOption("persist", "view_file_browser", False),
         config.TextOption("persist", "current_directory", ""),
+        config.IntOption("persist", "volume", "50"),
```

Comment:
I'd make this option more explicit: "mediaplayer_volume"
Reason: if we add more options related to the mediaplayer, we can "group" them using this common prefix.

Do not store the value of the slider position, but the value of the volume (both are the same for now, so the issue doesn't clearly appear).
If we change the way the volume is controlled we don't want to change the actual volume set.

Instead convert to/from to adjust UI control position, and read https://doc.qt.io/qt-5/qaudio.html#convertVolume

Reply:
Can we have an option mediaplayer_speed, default 1.0, which allows you to slow down or speed up the playing speed as some people (like me) like to preview music at a faster (or slower) speed than normal?

Reply:
@Sophist-UK I think it can be done later, in next pull requests. I want firstly add basic functional to the master branch.